### PR TITLE
[WIP] ch.swisstopo.geologie-tiefengeothermie_projekte update (model and mako adaptation)

### PR DESCRIPTION
**Pull request by alcapat**
Link with Jira: https://jira.swisstopo.ch/browse/BGDIDI_KB-774 (15.08.2018)

### CODE REVIEW

**Comment by alcapat** [comment]
Rq : In the model (stopo.py) we use table update (geologie-tiefengeothermie_projekte_update) with new delivery data for the test link. 
Just before going to prod, copy data on final table (geologie-tiefengeothermie_projekte) and remove changes on line 1130.

Test link : https://mf-chsdi3.int.bgdi.ch/ltalp_stopo_geologie-tiefengeothermie_projekte/shorten/7d1bcb953d

**Comment by AFoletti** [comment]
Why are the last two tooltip elements in bold?

**Comment by alcapat** [comment]
Link with mapfile_include PR : https://github.com/geoadmin/wms-mapfile_include/pull/324

File: chsdi/templates/htmlpopup/tiefengeothermie_projekte.mako [hunk 1]
```diff
@@ -20,10 +20,27 @@
...
+       <td>-</td>
+   % else:
+       <td><a href="${c['attributes']['download']}" target="_blank">Zip</a></td>
+   % endif
+   </tr>
+   <%
+   weblink = c['attributes']['weblink'].split('; ')
+   %>
+   <tr>
+     <th class="cell-left">${_('ch.swisstopo.geologie-tiefengeothermie_projekte.weblink')}</th>
```
Comment (AFoletti) [inline chsdi/templates/htmlpopup/tiefengeothermie_projekte.mako]: Please implement an "or "-"" for those cases where no link is present

Comment (alcapat) [reply chsdi/templates/htmlpopup/tiefengeothermie_projekte.mako]: Done

**Comment by AFoletti** [comment]
This LGTM now. Many thanks!

**Pull request merged**